Makefile

```diff
@@ -103,7 +105,8 @@ build-embeddings-arm:
 		-t text-embeddings-inference-arm
 	@cd ..
 	@rm -rf text-embeddings-inference
-setup-embeddings-arm:
+
+setup-embeddings-arm: guard-HUGGINGFACE_MODEL guard-CUSTOM_EMBEDDINGS_API_KEY
```

Comment:
Ah, the alternative is to export env vars from .env file within the make target!

But user needs to remember to set them.

Because I want this for momconnect I will just push it directly to main :'( Sorry for the mess

```
Ah! The alternative is this.
	@set -a && \
        source "$(CURDIR)/deployment/docker-compose/.core_backend.env" && \
        set +a && \
	docker run \
            ....
```

Reply:
Oh nooo it doesn't actually work. I've tested. Nvm.